src/adv.py

```diff
@@ -49,3 +77,41 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+
+# def moves(cmd):
+
+
+player_1 = Player("Rodean", room["outside"])
+
+while True:
+    print("current location:", player_1.current_room.name)
+    print(player_1.current_room.description)
+    player_cmd = input("Type n,s,e,w to move or q to quit:")
+    if player_cmd == "n":
+        if player_1.current_room.n_to:
+            print("you moved to the north")
+            player_1.current_room = player_1.current_room.n_to
```

Comment:
you'll want to add some code to check if `current_room` as a `.n_to` before attempting to move the player.